scripts/generateProptypes.ts

```diff
@@ -210,7 +210,6 @@ async function generateProptypes(
   const generatedForTypeScriptFile = sourceFile === tsFile;
   const result = ttp.inject(proptypes, sourceContent, {
     disablePropTypesTypeChecking: generatedForTypeScriptFile,
-    removeExistingPropTypes: true,
```

Comment:
Drive-by cleanup after https://github.com/mui-org/material-ui/pull/25193. This file is currently not type-checked in CI it seems.